File: PjblRa4/SorteadorItem.java
import java.util.ArrayList;
import java.util.List;

public class SorteadorItem {
    private List<String> itens;

    public SorteadorItem() {
        this.itens = new ArrayList<>();
    }

    public void adicionarItem(String item) {
        itens.add(item);
    }

    public String sortearItem() {
        if (itens.isEmpty()) {
            return "Nenhum item adicionado!";
        }
        int indice = (int) (Math.random() * itens.size());
        return "Item Sorteado: " + itens.get(indice);
    }
}

File: Ajuste/SorteadorItem.py
import random

class SorteadorItem:
    def __init__(self):
        self.itens = []

    def adicionar_item(self, item):
        self.itens.append(item)

    def sortear_item(self):
        if not self.itens:
            return "Nenhum item disponível para sorteio."
        item_sorteado = random.choice(self.itens)
        return f"Item Sorteado: {item_sorteado}"
